VERSION

```diff
@@ -1 +1 @@
-1.17.0
\ No newline at end of file
+1.18.0
```

Comment:
can we get rid of the newline? We cat this file in buildspec and I did not test with an extra newline

Reply:
Addressed in most recent commit. Will squash before merging. Took a couple tries but what finally got it was `perl -pi -e 'chomp if eof' VERSION`